src/filescanner.c

```diff
@@ -62,6 +62,7 @@
 #include "cache.h"
```

Comment:
I suggest placing library modules in a subdir. Like outputs (and the future inputs).